### ISSUE TITLE
If you don't mind I can start translating this to spanish

### ISSUE THREAD

**Issue by diazgilberto** [Avik-Jain/100-Days-Of-ML-Code#33]
**Is your feature request related to a problem? Please describe.**
I would like to start translating the content into Spanish and make this available to a larger community.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Avik-Jain** [comment]
yeah, you can I will make a repository just like the Chinese version and will add you as a contributor for the same and you can work there. 
just send me a few examples here.

**Comment by mablung69** [comment]
I also want to help with the translation into Spanish :D

**Comment by mablung69** [comment]
Ok! I start with day 2.

**Comment by Avik-Jain** [comment]
@diazgilberto @mablung69  Any updates or should I close this Issue?

**Comment by mablung69** [comment]
Not yet, i am actually working on the image template of Day 2, i am learning to modify image in this moment.

**Comment by Avik-Jain** [comment]
@diazgilberto  @mablung69  Any updates or should I close this Issue?

**Comment by mablung69** [comment]
Yes, i have day 2 ready

I working in day 3 and day 1 now.

![dia 2](https://user-images.githubusercontent.com/1727580/45935442-194b4700-bf9b-11e8-9583-011afd3d6fa2.jpg)

**Comment by diazgilberto** [comment]
@Avik-Jain I'm so sorry that I was the person the propose the translation but got caught up with work and school. I glad that @mablung69 is able to put some effort here.
